src/hooks/users.ts

```diff
@@ -85,6 +85,18 @@ export const useWorldUsersByIdWorkaround = () => {
   return { worldUsersById, isWorldUsersLoaded };
 };
 
+export const useWorldUserById = (id?: string) => {
+  useConnectWorldUsers();
+
+  const worldUsersById = useSelector(worldUsersByIdSelector) ?? {};
+
+  if (!id) return;
+
+  const user = worldUsersById[id];
+
+  return user;
+};
+
```

Comment:
Why is this being added here? On line 61 `useWorldUsersById` already exists, but it has a typing bug in it, and so you should be using `useWorldUsersByIdWorkaround` from line 77

```suggestion
```

Reply:
This new hook accepts user id and returns the user data if it exists in the worlds users object. It's just an extra step inside to avoid actually check for the user inside the component itself.

Reply:
Single use? If so just calle it useWorldUser or something. By I’d implies it’s a map, and having functions differ in name by a single s is error prone